### ISSUE TITLE
Diagnosis only tests TCP ports openings, not UDP

### ISSUE THREAD

**Issue by tituspijean** [YunoHost/issues#1914]
### Describe the bug

Diagnosis system only tests for TCP ports, not UDP. This leads to false positives whenever a service only needs a UDP port to be opened.

### Context

- Hardware: any
- YunoHost version: 4.x
- I have access to my server: any
- Are you in a special context or did you perform some particular tweaking on your YunoHost instance?: no

### Steps to reproduce

Case in the wild: https://github.com/YunoHost-Apps/wireguard_ynh/issues/32

### Expected behavior

Do not display an error if a port is only required to be open in UDP and TCP test fails.

**Comment by CodeShakingSheep** [comment]
Is this issue being worked on?

**Comment by alexAubin** [comment]
No, it's not really a big priority, and I don't even know if there's a straightforward way to test if an UDP port appears to be open, and even if there were, this implies non-trivial changes in the diagnosis system (and maybe app packaging to declare that the app is exposed on UDP instead of TCP), clearly we have more important things to work on

**Comment by ghost** [comment]
For what it is worth, I have TCP and UDP open for 8095 and I still get this error.

**Comment by zamentur** [comment]
Yesterday we discussed 2 solutions during inter is coding:

## Disable UDP test in remote diagnosis
We could simply disable UDP ports from remote diagnosis 
https://github.com/YunoHost/yunohost/blob/dev/src/diagnosers/14-ports.py#L47


## Use nftable to redirect UDP packet on a dedicated service

The idea is to setup a dedicated service on each yunohost instance and to start it before doing diagnosis, redirect with nftable all UDP packet accepted by nftable and coming from infra IP.

Like that we could detect that packets are received
